source/common/tracing/zipkin/zipkin_tracer_impl.cc

```diff
@@ -0,0 +1,194 @@
+#include "common/tracing/zipkin/zipkin_tracer_impl.h"
+
+#include "common/common/enum_to_int.h"
+#include "common/http/headers.h"
+#include "common/http/message_impl.h"
+#include "common/http/utility.h"
+#include "common/tracing/http_tracer_impl.h"
+#include "common/tracing/zipkin/zipkin_core_constants.h"
+
+#include "spdlog/spdlog.h"
+
+namespace Zipkin {
+
+ZipkinSpan::ZipkinSpan(Zipkin::Span& span) : span_(span) {}
```

Comment:
I only left `Zipkin::` in the `ZipkinSpan` class.  It turns out that, because `ZipkinSpan` inherits from `Tracing::Span`, any reference to `Span` is considered to be a reference to `Tracing::Span` and not to `Zipkin::Span`.  I explicitly refer to `Zipkin::Span` to disambiguate. I was surprised by this behavior. Any suggestion?